deployment/debian-x64/Dockerfile

```diff
@@ -11,12 +13,11 @@ RUN apt-get update \
  && chown root:root /etc/apt/sources.list.d/microsoft-prod.list \
  && apt-get update
 
-WORKDIR /repo
+WORKDIR ${SOURCEDIR}
 COPY . .
+COPY ./deployment/debian-x64/pkg-src ./debian
 
-RUN yes|mk-build-deps -i \
- && dpkg-buildpackage -us -uc \
- && mkdir /dist \
- && mv /jellyfin*deb /dist
+RUN yes | mk-build-deps -i debian/control \
+    && dpkg-buildpackage -us -uc
```

Comment:
Another case for splitting out the binary build from the package build: it's actually totally redundant to do ` dotnet publish` and copy the files into the docker image, only to run `dpkg-buildpackage` which rebuilds it (and *must* do so).

Reply:
Not everyone might want a *.deb package though. I mean I guess they could then grab the linux-x64 package?

Reply:
Right but we need a specific build setup for the Deb package in particular, separate from the others, so I can build it consistently. I don't like the overlap is all, I'd rather have a generic Linux binary, then a Debian package that does zero `dotnet` work on the current machine (only in Docker like the current script).

Reply:
Same idea when we integrate the RPM package, it would have a totally separate folder and a totally separate build process, also inside Docker.

Reply:
Well the debian package is completely separate from the portable debian build, they are just in the same script after one another. I can pull those apart if you want.

Reply:
Yes please, let's keep it totally separate: `debian-package-x64` folder that only does the packages (src + binary), then a `debian-x64` that does the portable build.